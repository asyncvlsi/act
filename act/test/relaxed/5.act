template <pint x | pint y>
defproc foo (bool a, b, c)
{
  [ x > 3 ->
  prs {
   (:i:y: a => b- )
  }
  ]
}


foo<4,3> f[3];

foo<4,4> f[4..5];

f[5] = f[4];

f[2] = f[4];
