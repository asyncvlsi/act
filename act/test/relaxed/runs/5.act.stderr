Typechecking failed during connection.
Type 1: defproc <4,3> (bool a; bool b; bool c)
Type 2: defproc <4,4> (bool a; bool b; bool c)
	File `5.act', line: 18, col: 2
